tests/test_notebooks/test_notebooks.py

```diff
@@ -25,6 +26,7 @@ def force_rewrite_comparator(expected: str, actual: str) -> str:
     return msg
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Figure out charset fail on windows in github actions")
```

Comment:
Is there an issue for the bug that you can link to? I think there should be if tests are skipped although a problem still exists.

Reply:
@dalito Great point. I found this issue that seems appropriate. https://github.com/linkml/linkml/issues/314

The strategy I'd suggest is to mark this PR (2488) complete for review (I'm past the suggested 5-hour december data day time)

Then I can update 314 with info about the tests still writing and modifying notebooks/tmp and the skip condition because those things seem more in the 314 scope.

Please let me know if you'd like me to refer to issue 314 in the skip reason.

Reply:
Yes, referring to #314 would be helpful to understand the skip.

Reply:
Thanks, @dalito I updated the reason and commented Issue 314 with the specific failure message.